src/main/java/seedu/contax/logic/parser/ImportCsvParser.java

```diff
@@ -84,4 +64,14 @@ private void checkDuplicateAndAdd(HashSet<Integer> hashSet, int position) throws
         }
     }
 
+    /**
+     * Checks if the prefix exists in the command and provides the index, otherwise uses the default index.
+     */
+    private int prefixChecker(int defaultValue, Prefix prefix, ArgumentMultimap argsMultimap) throws ParseException {
```

Comment:
Likewise, method should be named a verb instead of a noun.

Reply:
thanks for the catch, updated in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/248/commits/2b67c04764a53ad085721a659134752196fea472